pkg/apis/crds/karpenter.azure.com_aksnodeclasses.yaml

```diff
@@ -53,9 +53,6 @@ spec:
                 - Ubuntu2204
                 - AzureLinux
                 type: string
-              imageVersion:
-                description: ImageVersion is the image version that instances use.
```

Comment:
what is the plan for replacing it?

Reply:
https://github.com/Azure/karpenter-provider-azure/pull/526 relies on this field so just removing it feels wrong? WIll we no longer drift on imageVersion? whats the new upgrade plan